src/libraries/System.Text.Json/tests/JsonStringTests.cs

```diff
@@ -110,6 +110,25 @@ public static void TestDateTimeOffset(string testStr, string expectedStr)
             Assert.Equal(dateTimeOffset, dateTimeOffset2);
         }
 
+        [Theory]
+        [MemberData(nameof(JsonDateTimeTestData.ValidISO8601Tests), MemberType = typeof(JsonDateTimeTestData))]
+        public static void TestDateTimeAndDateTimeOffsetWithEscapedChars(string testStr, string iso8601Str)
+        {
+            var trimmedTestStr = testStr.Trim('"');
```

Comment:
nit: Don't use `var` unless the right hand side has `new`, explicit cast, or otherwise, obvious what the type is (while statically reading).